### ISSUE TITLE
`chain get` no longer works

### ISSUE THREAD

**Issue by Kubuxu** [filecoin-project/lotus#7701]
### Checklist

- [X] This is **not** a security-related bug/issue. If it is, please follow please follow the [security policy](https://github.com/filecoin-project/lotus/security/policy).
- [X] This is **not** a question or a support request. If you have any lotus related questions, please ask in the [lotus forum](https://github.com/filecoin-project/lotus/discussions).
- [X] This is **not** a new feature request. If it is, please file a [feature request](https://github.com/filecoin-project/lotus/issues/new?assignees=&labels=need%2Ftriage%2Ckind%2Ffeature&template=feature_request.yml) instead.
- [X] This is **not** an enhancement request. If it is, please file a [improvement suggestion](https://github.com/filecoin-project/lotus/issues/new?assignees=&labels=need%2Ftriage%2Ckind%2Fenhancement&template=enhancement.yml) instead.
- [X] I **have** searched on the [issue tracker](https://github.com/filecoin-project/lotus/issues) and the [lotus forum](https://github.com/filecoin-project/lotus/discussions), and there is no existing related issue or discussion.
- [X] I am running the [`Latest release`](https://github.com/filecoin-project/lotus/releases), or the most recent RC(release canadiate) for the upcoming release or the dev branch(master), or have an issue updating to any of these.
- [X] I did not make any code changes to lotus.

### Lotus component

- [ ] lotus daemon - chain sync
- [ ] lotus miner - mining and block production
- [ ] lotus miner/worker - sealing
- [ ] lotus miner - proving(WindowPoSt)
- [ ] lotus miner/market - storage deal
- [ ] lotus miner/market - retrieval deal
- [ ] lotus miner/market - data transfer
- [ ] lotus client
- [ ] lotus JSON-RPC API
- [ ] lotus message management (mpool)
- [X] Other

### Lotus Version

```text
Daemon:  1.13.2-dev+mainnet+git.e196a81e9.dirty+api1.4.0
Local: lotus version 1.13.2-dev+mainnet+git.e196a81e9.dirty
```

### Describe the Bug

Calling `lotus chain get` returns "empty" output.
```
./lotus chain get  bafy2bzacebcd4btqq6zgelu6dmnqd5qtoj2kttdtwkqsespkxfmu35fswn2i4
{
        "Block": {},
        "Node": {}
}
```

### Logging Information

```text
<no related log output>
```

### Repo Steps

Call `lotus chain get  bafy2bzacebcd4btqq6zgelu6dmnqd5qtoj2kttdtwkqsespkxfmu35fswn2i4` which is one of the states of the market actor. but calling `lotus chain get /pstate` also reproduces it.

**Comment by jennijuju** [comment]
the bug is introduced by v1.13.1 and should be in `ChainGetNode`.

**Comment by jennijuju** [comment]
`go get: downgraded github.com/ipfs/go-merkledag v0.4.1 => v0.3.2` fix it. the dep upgrade was introduced by https://github.com/filecoin-project/lotus/commit/8b01725129bf005b826e7c238b2c1301748db814

**Comment by ribasushi** [comment]
@willscott @warpfork ☝️ does this ring a bell?
`cbor map[cid:xxx node_type:cbor]` with `v0.3.2`
vs
`block map[block:]` with `v0.4.1`

**Comment by willscott** [comment]
cc on @hannahhoward for connecting the dots on the ipld-prime update that happened in that bump of `go-merkledag`.

**Comment by mur-me** [comment]
Hey @jennijuju , will this issue be included in the upcoming 1.13.2 release?

**Comment by jennijuju** [comment]
https://github.com/ipfs/go-ipld-legacy/pull/7 should fix it.

**Comment by jennijuju** [comment]
> Hey @jennijuju , will this issue be included in the upcoming 1.13.2 release?

yes
